db/migrate/20220727123030_create_articles.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateArticles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :articles, id: :uuid do |t|
+      t.string :message
+      t.references :user
```

Comment:
As I see from your Article model you force this field presence. So maybe you add `null: false` option here?

I would also recommend you reading this https://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference and providing all the options explicitly so that you can learn them :)